.github/actions/check-disk-space/action.yml

```diff
@@ -56,6 +56,9 @@ runs:
           # Stop the daemon and launchctl will automatically start it again, thus accomplish a restart and free up the above file
           sudo launchctl stop com.apple.coresymbolicationd || true
 
+          # Clean up crash reports on the runner
+          sudo rm -rf "/System/Volumes/Data/Library/Logs/CrashReporter" || true
```

Comment:
Do we have a way to preserve in case we need to investigate?

Reply:
Ideally, we should keep some reports to help with the investigation, but we haven't look into this before in the past.  I will take this example as an exercise to see if there is any useful information there.  

On the one hand, if we need to submit the issue to Apple experts, then it's useful to have the crash reports.  On the other hand, cleaning up the crash reports needs to be done to keep the CI sanity.  Let me see if I can archive the folder and keep it in `/tmp` instead (depending on how big the archive it, the raw reports took a total of 30GB)